src/index.css

```
@@ -109,6 +80,6 @@ All colors MUST be HSL.
 
   body {
     @apply bg-background text-foreground;
-    font-family: 'Noto Serif Devanagari', serif;
+    font-family: Helvetica, Arial, 'Noto Sans Devanagari', sans-serif;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)